### PR TITLE
Set commentstring for nu filetype

### PR DESCRIPTION
**Pull request by mrcjkb**

### CODE REVIEW

File: runtime/ftplugin/nu.vim [hunk 1]
```diff
@@ -0,0 +1,13 @@
+" Vim filetype plugin
+" Language:	nu
```
Comment (dkearns) [inline runtime/ftplugin/nu.vim]:
```suggestion
" Language:	Nu
```

I do what I can...

**Review by dkearns — CHANGES_REQUESTED** [review]

**Comment by chrisbra** [comment]
Thanks. Please also add yourself to the Maintainers file.

**Comment by chrisbra** [comment]
I read https://programming-nu.github.io/syntax 
Is  `#` prefered as leading comment character (instead of using `;`)?

**Comment by mrcjkb** [comment]
> I read https://programming-nu.github.io/syntax Is `#` prefered as leading comment character (instead of using `;`)?

I based this on [what they did in tree-sitter-nu](https://github.com/nushell/tree-sitter-nu/blob/0bb9a602d9bc94b66fab96ce51d46a5a227ab76c/plugin/init.lua#L5).

**Comment by chrisbra** [comment]
alright thanks

**Pull request closed without merging**